vulyk/assets.py

```diff
@@ -1,9 +1,13 @@
-# coding=utf-8
+# -*- coding: utf-8 -*-
+
 from flask.ext.assets import Environment, Bundle
 
 
 def init(app):
+    """ Bundle projects assets """
+
```

Comment:
Probably, it's better way to think about plugin's assets at #10 and #12. Event, especially on #12, as it will be much easier to test and wee what comes as a result. That is why I'd like to move out managing assets for now from this diff, and start working on #12 to be able to test all in place